test/SerialIOTest.cpp

```diff
@@ -83,6 +83,54 @@ TEST_CASE( "multi_series_test", "[serial]" )
     allSeries.clear();
 }
 
+TEST_CASE( "multiple_series_handles_test", "[serial]" )
+{
+    /*
+     * First test: No premature flushes through destructor when another copy
+     * is still around
+     */
+    {
+        std::unique_ptr< openPMD::Series > series_ptr;
+        {
+            openPMD::Series series(
+                "sample%T.json", openPMD::AccessType::CREATE );
+            series_ptr = std::unique_ptr< openPMD::Series >(
```

Comment:
just for modern C++, `new`-free style:
```suggestion
            series_ptr = std::make_unique< openPMD::Series >( series );
```

Reply:
Oo, I keep forgetting we're allowed to use C++14 now :)

Reply:
Oh, we did not officially switch yet - dang I always forget this is missing in C++11.
Soon - how far is PIConGPU in `dev`?

Reply:
Ah right, I confused both of them. PIConGPU is now running on C++14. So I'm keeping the raw constructors for now?

Reply:
Problem solved - you can now use C++14 #825 :)

Reply:
Nice, updated